File: Code/dotNetBasics/ConsoleApp/ADStack/Components/LidarClusteringForPedestrian.cs
using ConsoleApp.ADStack.Models;

namespace ConsoleApp.ADStack.Components
{
    /// <summary>
    /// Detect pedestrian with our amazing clustering algo
    /// </summary>
    internal class LidarClusteringForPedestrian : IComponent
    {
        public void Start()
        {
            Console.WriteLine("[LidarClusteringForPedestrian] Starting.");
        }

        public void Stop()
        {
            Console.WriteLine("[LidarClusteringForPedestrian] Stopping.");
        }
    }
}

File: Code/Fluent.Java/src/adstack/components/LidarClusteringForPedestrian.java
package adstack.components;

import adstack.models.IComponent;

public class LidarClusteringForPedestrian implements IComponent {

	@Override
	public void start() {
        System.out.println("[LidarClusteringForPedestrian] Starting.");
	}

	@Override
	public void stop() {
        System.out.println("[LidarClusteringForPedestrian] Stopping.");
	}

}
